group-aws/integration.md

```diff
@@ -12,167 +12,28 @@ shall be considered a feature request.
 Following the 2.5 release, all new modules MUST have integration
 tests.
 
-Modules marked `preview` SHOULD have integration tests for new
-features.
+Following the 2.10 release, all modules MUST have integration tests.
 
 Bug fixes for modules that currently have integration tests SHOULD
 have tests added. Depending upon the urgency of the bug fix, we may
 accept bug fixes without tests.
 
 ## Expected test criteria
 
-* Resource creation
-* Resource creation again
 * Resource creation under check mode
+* Resource creation
+* Resource creation again (idempotency) under check mode
+* Resource creation again (idempotency)
+* Resource modification under check mode
 * Resource modification
+* Resource modification again (idempotency) under check mode
+* Resource modification again (idempotency)
+* Resource deletion under check mode
 * Resource deletion
+* Resource deletion (of a non-existent resource) under check mode
+* Resource deletion (of a non-existent resource)
 
```

Comment:
I think somewhere we had some wording about running through the 4-step resource modification cycle for each parameter (where appropriate)

Reply:
What is the bit in https://github.com/ansible-collections/amazon.aws/blob/main/CONTRIBUTING.md?  I'll add that in.